Document https://github.com/jreleaser/jreleaser/issues/382

andreabettich opened the pull request:
<!-- Please target the `development` branch -->

<!--- The issue this PR addresses -->
Fixes https://github.com/jreleaser/jreleaser/issues/382

Is it clear what the new configuration does?

aalmiray commented on docs/modules/configuration/partials/release/json/common-head.adoc:
```diff
@@ -58,6 +58,12 @@ endif::gitservice_api[]
+      // May define a `JRELEASER_PREVIOUS_TAG_NAME` environment variable instead.
+      // If left unspecified, will use the previous logical tag of `tagName`.
+      // icon:dot-circle[] icon:eye-slash[]
+      previousTagName = "v0.9.0"
```
Missing a `,` at the end

andreabettich commented:
fixed the JSON snippet, mentioned the env `JRELEASER_PREVIOUS_TAG_NAME ` and rebased unto development and changed the target.

Thanks a lot for your time!

aalmiray commented:
Thanks!
